srcpkgs/purescript/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'purescript'
+pkgname=purescript
+version=0.14.0
+revision=1
+wrksrc="purescript-${version}"
+build_style=haskell-stack
+makedepends="ncurses-libtinfo-devel ncurses-devel zlib-devel"
```

Comment:
Apparently, the libtinfo dependency has been removed (see https://github.com/purescript/purescript/pull/4069). The change still hasn't been released though.